plugins/ruby-foreman-azure/debian/gem.list

```diff
@@ -1,5 +1,5 @@
 # list of gem urls to download via Jenkins, space separated
-GEMS="https://rubygems.org/downloads/foreman_azure-1.0.1.gem"
+GEMS="https://rubygems.org/downloads/foreman_azure-1.1.0.gem"
 GEMS="$GEMS https://rubygems.org/downloads/fog-azure-0.0.2.gem"
 GEMS="$GEMS https://rubygems.org/downloads/azure-0.7.5.gem"
```

Comment:
Would you mind updating this to 0.7.7? It should fix the error seen in #1368 (which may recur here) and to remove the uuid/macaddr gems.